app/views/microposts/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: micropost, local: true) do |form| %>
+  <% if micropost.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(micropost.errors.count, "error") %> prohibited this micropost from being saved:</h2>
+
+      <ul>
+      <% micropost.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= form.label :content %>
+    <%= form.text_area :content, id: :micropost_content %>
+  </div>
+
+  <div class="field">
+    <%= form.label :user_id %>
+    <%= form.number_field :user_id, id: :micropost_user_id %>
```

Comment:
if you write this part like here. 

```
<%= form.number_field :testtest_id, id: :micropost_user_id %>
```

You will get an error.
Why does the error show up?